packages/indy-vdr/src/IndyVdrApi.ts

```diff
@@ -13,7 +13,7 @@ import { multiSignRequest, signRequest } from './utils/sign'
 @injectable()
 export class IndyVdrApi {
   private agentContext: AgentContext
-  private indyVdrPoolService: IndyVdrPoolService
+  public indyVdrPoolService: IndyVdrPoolService
```

Comment:
We shouldn't make indyVdrPoolService public, but rather add the methods we want to expose to this class as well

Reply:
okay got it